websites/W/Wiki.gg/presence.ts

```diff
@@ -2,75 +2,123 @@ const presence = new Presence({
 		clientId: "978186598669758504",
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
-let title: string, wikiTitle: string;
+let title: string;
+enum Assets {
+	Logo = "https://i.imgur.com/4YN40As.png",
+	Search = "https://i.imgur.com/oGQtnIY.png",
+	Read = "https://i.imgur.com/wPUmqu5.png",
+}
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			browse: "general.browsing",
+			search: "general.searchFor",
+			viewHome: "general.viewHome",
+			buttonViewPage: "general.buttonViewPage",
+			buttonViewProfile: "general.buttonViewProfile",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+
+function capitalizeFirstLetter(string: string) {
+	if (!string) return "Undefined";
+	return (
+		string.trim().charAt(0).toUpperCase() + string.trim().slice(1).toLowerCase()
+	);
```

Comment:
```suggestion
	return string.trim().charAt(0).toUpperCase() + string.trim().slice(1).toLowerCase();
```
cant it just be this?

Reply:
idem dito